2023-03-02 12:47:14,950 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Configure stats pid to 22164
2023-03-02 12:47:14,951 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Loading settings from C:\Users\anant\.config\wandb\settings
2023-03-02 12:47:14,952 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Loading settings from C:\Users\anant\CS6910\Assignment 1\wandb\settings
2023-03-02 12:47:14,952 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-02 12:47:14,952 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-02 12:47:14,952 INFO    MainThread:22164 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\anant\CS6910\Assignment 1\wandb\run-20230302_124714-ev8yamhs\logs\debug.log
2023-03-02 12:47:14,954 INFO    MainThread:22164 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\anant\CS6910\Assignment 1\wandb\run-20230302_124714-ev8yamhs\logs\debug-internal.log
2023-03-02 12:47:14,954 INFO    MainThread:22164 [wandb_init.py:init():532] calling init triggers
2023-03-02 12:47:14,954 INFO    MainThread:22164 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {'epochs': 10, 'batch_size': 32, 'learning_rate': 0.001, 'activation': 'sigmoid_function', 'optimizer': 'adam', 'weight_init': 'random_initialization', 'lambd': 0, 'neurons': 784, 'num_hidden_layers': 1, 'beta1': 0.9, 'beta2': 0.99, 'beta': 0.9, 'momentum': 0.9, 'loss': 'cross_entropy_function'}
2023-03-02 12:47:14,954 INFO    MainThread:22164 [wandb_init.py:init():554] re-initializing run, found existing run on stack: ygz1l98a
2023-03-02 12:47:14,955 INFO    MainThread:22164 [wandb_run.py:_finish():1854] finishing run ananthu2014/CS6910-Assignment 1/ygz1l98a
2023-03-02 12:47:14,955 INFO    MainThread:22164 [jupyter.py:save_history():449] not saving jupyter history
2023-03-02 12:47:14,955 INFO    MainThread:22164 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-02 12:47:14,955 INFO    MainThread:22164 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-02 12:47:14,955 INFO    MainThread:22164 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-02 12:47:14,955 INFO    MainThread:22164 [wandb_run.py:_restore():2066] restore
2023-03-02 12:47:14,955 INFO    MainThread:22164 [wandb_run.py:_restore():2072] restore done
2023-03-02 12:47:22,268 INFO    MainThread:22164 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-02 12:47:22,268 INFO    MainThread:22164 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-02 12:47:22,277 INFO    MainThread:22164 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-02 12:47:22,285 INFO    MainThread:22164 [wandb_init.py:init():588] starting backend
2023-03-02 12:47:22,285 INFO    MainThread:22164 [wandb_init.py:init():592] setting up manager
2023-03-02 12:47:22,285 INFO    MainThread:22164 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-03-02 12:47:22,294 INFO    MainThread:22164 [wandb_init.py:init():599] backend started and connected
2023-03-02 12:47:22,321 INFO    MainThread:22164 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-02 12:47:22,321 INFO    MainThread:22164 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-02 12:47:22,321 INFO    MainThread:22164 [wandb_init.py:init():687] updated telemetry
2023-03-02 12:47:22,405 INFO    MainThread:22164 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-02 12:47:24,129 INFO    MainThread:22164 [wandb_run.py:_on_init():2134] communicating current version
2023-03-02 12:47:24,204 INFO    MainThread:22164 [wandb_run.py:_on_init():2143] got version response 
2023-03-02 12:47:24,204 INFO    MainThread:22164 [wandb_init.py:init():775] starting run threads in backend
2023-03-02 12:47:40,113 INFO    MainThread:22164 [wandb_run.py:_console_start():2114] atexit reg
2023-03-02 12:47:40,113 INFO    MainThread:22164 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-02 12:47:40,113 INFO    MainThread:22164 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-02 12:47:40,113 INFO    MainThread:22164 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-02 12:47:40,113 INFO    MainThread:22164 [wandb_init.py:init():817] run started, returning control to user process
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Configure stats pid to 22164
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Loading settings from C:\Users\anant\.config\wandb\settings
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Loading settings from C:\Users\anant\CS6910\Assignment 1\wandb\settings
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\anant\CS6910\Assignment 1\wandb\run-20230302_125605-uc1wi25p\logs\debug.log
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\anant\CS6910\Assignment 1\wandb\run-20230302_125605-uc1wi25p\logs\debug-internal.log
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000013120EC2E50>
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_init.py:init():532] calling init triggers
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-02 12:56:05,404 INFO    MainThread:22164 [wandb_init.py:init():554] re-initializing run, found existing run on stack: ev8yamhs
2023-03-02 12:56:05,420 INFO    MainThread:22164 [wandb_run.py:_finish():1854] finishing run ananthu2014/CS6910-Assignment 1/ev8yamhs
2023-03-02 12:56:05,420 INFO    MainThread:22164 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-02 12:56:05,420 INFO    MainThread:22164 [wandb_run.py:_restore():2066] restore
2023-03-02 12:56:05,420 INFO    MainThread:22164 [wandb_run.py:_restore():2072] restore done
